### ISSUE TITLE
Missing option "auto-extract to source dir"

### ISSUE THREAD

**Issue by bsto** [activescott/lessmsi#94]
Currently users are always forced to select the target dir for extracted stuff.

This is mostly unnecessary and tedious.

I miss an options "automatically extract files to source dir" ...which is used immediately after clicking on "extract"

**Comment by activescott** [comment]
Makes sense to me! Thanks for the suggestion. I'll make it so upon opening a new .msi file it will default the extraction directory/textbox to the same directory as the .msi file.
